PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -225,7 +225,11 @@ void ScintillaEditView::init(HINSTANCE hInst, HWND hPere)
 {
 	if (!_hLib)
 	{
-		throw std::runtime_error("ScintillaEditView::init : SCINTILLA ERROR - Can not load the dynamic library");
+		_hLib = loadSciLexerDll();

```

Comment:
Is the function called only once? If so, the first `if (!_hLib)` can probably go.

Reply:
Sadly it's called at least twice:

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/b01a3966681cacdfb1fce8313822c234566d98f3/PowerEditor/src/Notepad_plus.cpp#L222

and also for each plugin requested Scintilla:

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/b01a3966681cacdfb1fce8313822c234566d98f3/PowerEditor/src/ScitillaComponent/ScintillaCtrls.cpp#L36

It's what makes refactoring the code difficult :/